Readme.md

```diff
@@ -32,35 +32,38 @@ Researchers can compose their own surveys with custom images and sounds. The sur
 
 On the client side, test subjects just need to run a HTML5 capable browser.
 
-On the server side, Coloring Book is a WSGI application, which means that it will run on any webserver that supports standard Python web applications. Apart from a WSGI-enabled webserver, install the dependencies using `pip install -r requirements.txt` or preferably `pip-sync` from the pip-tools package.
+Coloring Book is deployed using Docker Compose. The application needs two configuration files. 
 
-Assuming you opt to use MySQL (or anything other than SQLite), you need to create an empty database with pre-configured access rights for the Coloring Book application to use. For example by entering the following commands while running `mysql` as root user:
+Docker needs a file called `.env` to be present in the same folder as `docker-compose.yml`, containing at least the following settings.
 
-    create database coloringbook;
-    grant all privileges on coloringbook.* to 'coloringbook'@'localhost' identified by 'myawesomepassword';
+    MYSQL_HOST=abcdefg
+    MYSQL_PORT=1234
+    MYSQL_USER=abcdefg
+    MYSQL_PASSWORD=abcdefg
+    MYSQL_DB=abcdefg
+    MYSQL_ROOT_PASSWORD=abcdefg
 
-**Note** that you should not use the password quoted above. You can choose the name of the database and the name of the user freely as well.
-
-Coloring Book obtains the necessary information to connect to the database from a user-provided configuration file. This is what the file contents should minimally look like:
+Secondly, a file `config.py` should be created in the `colouringbook` package folder with minimally the following contents.
```

Comment:
The confusing suggestion to put the config file in the root directory has been removed.